dbt_checkpoint/check_model_has_constraints.py

```diff
@@ -0,0 +1,119 @@
+import argparse
+import os
+import time
+from typing import Any, Dict, Optional, Sequence
+
+from dbt_checkpoint.tracking import dbtCheckpointTracking
+from dbt_checkpoint.utils import (
+    JsonOpenError,
+    add_default_args,
+    get_dbt_manifest,
+    get_missing_file_paths,
+    get_model_sqls,
+    get_models,
+    Model,
+    ParseJson,
+)
+
+
+def is_equal(constraint:Dict[str, Any], model_constraint:Dict[str, Any]) -> bool:
+    for key, value in constraint.items():
+        if key not in model_constraint or model_constraint[key] != value:
+            return False
+    return True
+
+
+def has_constraints(constraints:Sequence[Dict[str, Any]], model:Model) -> bool:
+    model_constraints = model.node.get("constraints", [])
+    for constraint in constraints:
+        found = False
+        for model_consraint in model_constraints:
```

Comment:
`model_consraint` seems to have a typo (no `t`)

Reply:
fixed